src/lib/README.md

```diff
@@ -0,0 +1,16 @@
+# Lib
+
+The lib folder contains a variety of Express middleware functions, designed to interact with the [Express App](https://expressjs.com/en/4x/api.html#app)s directly, or to work with the `req`/`res`/`next` pattern of [Express middleware](https://expressjs.com/en/guide/writing-middleware.html).
+
+- [`i18next`](./i18next) - Express router module for configuring [i18next](https://www.i18next.com/) for use with CRI paths and templates
+- [`axios`](./axios.js) - shared Axios client with support for [`x-forwarded-for`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For) and `x-scenario-id` headers
+- [`error-handling`](./error-handling.js) - global error handler configured to redirect to initial caller of CRI
+- [`headers`](./headers.js) - patch for load balancer forwarding incorrect header value for `x-forwarded-proto`
+- [`helmet`](./helmet.js) - default [Helmet](https://helmetjs.github.io/) configuration
+- [`locals`](./locals.js) - helpers for assigning [res.local](https://expressjs.com/en/api.html#res.locals) values from [req.app](https://expressjs.com/en/api.html#req.app) settings
+- [`oauth`](./oauth.js) - shared Axios client
```

Comment:
Do you think there's anything more we can add to this description? Looking in `oauth.js` it's not using axios at all so I'm struggling to make the connection

Reply:
That was a copy-paste bug from line 6 - that's now been fixed.